README.md

```diff
@@ -1,16 +1,21 @@
 
 # Axe + Azure Pipelines: Automate accessibility testing in your CI builds
 
-Samples demonstrating how to use [axe](https://www.deque.com/axe/) for automated accessibility testing of websites in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds.
-
-Automated accessibility tests can detect some common accessibility problems such as missing or invalid properties. But most accessibility problems can only be discovered through manual testing. We recommend [Accessibility Insights](https://accessibilityinsights.io/docs/en/web/overview), a free and open source dev tool that walks you though assessing a website for 100% [WCAG 2.0 AA](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview&versions=2.0&levels=aaa) coverage.
+This project contains sample projects (see next section [Available samples](#available-samples)) demonstrating how to implement automated accessibility testing in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds using [axe-core](https://github.com/dequelabs/axe-core), the same accessibility scanning engine used in [Accessibility Insights for Web](https://github.com/microsoft/accessibility-insights-web).
 
 ## Available samples
 
-* [typescript-selenium-webdriver](./typescript-selenium-webdriver/README.md): Uses [TypeScript](https://typescriptlang.org), [Jest](https://jestjs.io/), [Selenium](https://www.seleniumhq.org), and [axe-webdriverjs](https://www.npmjs.com/package/axe-webdriverjs) to test a web page for accessibility issues. Shows how to baseline existing failures with [Jest Snapshot Testing](https://jestjs.io/docs/en/snapshot-testing) and how to visualize failures in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) using [SARIF](https://sarifweb.azurewebsites.net/).
+The following sample projects specify the main technologies used. A team that uses comparable tools and frameworks should be able to refer to the sample and update their existing test to incorporate automated accessibility checks.
+
+* **[Sample 1: typescript-selenium-webdriver-sample](./typescript-selenium-webdriver/README.md)**: 
+  * Useful for teams using technologies similar to TypeScript, Jest, Selenium, and axe-webdriverjs. 

```

Comment:
I worry this might be a little *too* permissive; Puppeteer and Cypress are technologies "similar to Selenium", but it might be confusing for a project using them to go down this sample...

Reply:
What if we said "Useful for teams using Selenium and TypeScript/JavaScript" instead? Would that make it too restrictive?

Reply:
I think that sounds about right to me